step-01/js/main.js

```diff
@@ -1,26 +1,73 @@
 'use strict';
 
-navigator.getUserMedia = navigator.getUserMedia ||
-    navigator.webkitGetUserMedia || navigator.mozGetUserMedia;
+// Define navigator.mediaDevices.getUserMedia() for older devices.
 
-var constraints = {
+// Old browsers might not implement mediaDevices, so we set an empty object.
```

Comment:
Use second person throughout, not first person: 'you' not 'we'.

(This is standard Google style, oriented to action by the reader — 'you do this' not 'we do this' — and to avoid the 'professorial' voice.)

Reply:
Done.